api/web/base.py

```diff
@@ -27,6 +27,11 @@ def __init__(self, request=None, response=None): # pylint: disable=super-init-no
         """Set uid, source_site, public_request, and superuser"""
         self.initialize(request, response)
 
+        # If user is attempting to log in through `/login`, ignore Auth here:
+        # In future updates, move login and logout handlers to class that overrides this init
+        if self.request.path == 'api/login':
+            return
```

Comment:
Legacy wouldn't be the right word. This logic is used to AuthX a request when the header contains auth information (session token/drone info/API key). For login/out endpoints, we should bypass this logic as we aren't trying to evaluate the authx via the headers.